src/components/Navbar/BoardLabel.vue

```diff
@@ -0,0 +1,50 @@
+<template>
+  <li class="board-label">
+    <svg
+      v-if="!name.includes('Hide')"
+      width="16"
+      height="16"
+      xmlns="http://www.w3.org/2000/svg"
+      class="scale-[.8] min-[350px]:scale-100"
+    >
+      <path
+        d="M0 2.889A2.889 2.889 0 0 1 2.889 0H13.11A2.889 2.889 0 0 1 16 2.889V13.11A2.888 2.888 0 0 1 13.111 16H2.89A2.889 2.889 0 0 1 0 13.111V2.89Zm1.333 5.555v4.667c0 .859.697 1.556 1.556 1.556h6.889V8.444H1.333Zm8.445-1.333V1.333h-6.89A1.556 1.556 0 0 0 1.334 2.89V7.11h8.445Zm4.889-1.333H11.11v4.444h3.556V5.778Zm0 5.778H11.11v3.11h2a1.556 1.556 0 0 0 1.556-1.555v-1.555Zm0-7.112V2.89a1.555 1.555 0 0 0-1.556-1.556h-2v3.111h3.556Z"
+      />
+    </svg>
+    <svg
+      v-else
+      width="18"
+      height="16"
+      xmlns="http://www.w3.org/2000/svg"
+      class="scale-[.8] min-[350px]:scale-100"
+    >
+      <path
+        d="M8.522 11.223a4.252 4.252 0 0 1-3.654-5.22l3.654 5.22ZM9 12.25A8.685 8.685 0 0 1 1.5 8a8.612 8.612 0 0 1 2.76-2.864l-.86-1.23A10.112 10.112 0 0 0 .208 7.238a1.5 1.5 0 0 0 0 1.524A10.187 10.187 0 0 0 9 13.75c.414 0 .828-.025 1.239-.074l-1-1.43A8.88 8.88 0 0 1 9 12.25Zm8.792-3.488a10.14 10.14 0 0 1-4.486 4.046l1.504 2.148a.375.375 0 0 1-.092.523l-.648.453a.375.375 0 0 1-.523-.092L3.19 1.044A.375.375 0 0 1 3.282.52L3.93.068a.375.375 0 0 1 .523.092l1.735 2.479A10.308 10.308 0 0 1 9 2.25c3.746 0 7.031 2 8.792 4.988a1.5 1.5 0 0 1 0 1.524ZM16.5 8a8.674 8.674 0 0 0-6.755-4.219A1.75 1.75 0 1 0 12.75 5v-.001a4.25 4.25 0 0 1-1.154 5.366l.834 1.192A8.641 8.641 0 0 0 16.5 8Z"
+      />
+    </svg>
+    <div class="flex items-center gap-[2px]">
+      <span v-if="name.includes('Create')">+</span>
+      <span class="boards-name">{{ name }}</span>
+    </div>
+  </li>
+</template>
+
+<script setup lang="ts">
+defineProps<{
+  name: string
+}>()
+</script>
+
+<style scoped>
+.board-label {
+  @apply flex items-center gap-2 min-[350px]:gap-4 h-12 w-[90%] px-3 min-[350px]:px-6;
+  @apply rounded-r-[100px] cursor-pointer sm:pr-11;
+  @apply hover:bg-purple-320 hover:text-purple-400 hover:fill-purple-400;
+  @apply focus:bg-purple-320 focus:text-purple-400 focus:fill-purple-400;
+  @apply dark:hover:bg-white dark:focus:bg-white transition-colors duration-300;
```

Comment:
Not sure what you mean?

If this is about this empty space after `sm:pr-11`, it's because I like to group the Tailwind classess (when being separated for several lines like here), so each of the lines contains similar/related properties. As such, when I need to change something, e.g. the hover and focus states, I know all hover-related classess are in one line and the focus-related classess are in the other line and I don't have to read everything to be able to find what I'm after.

Reply:
Extra space means either your indentation is wrong or there are two spaces between blocks of text and one should be deleted. I think it's ok now anyway.